server/routes/partyRoutes.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import PartyController from '../controllers/partyController';
+import PartyMiddleware from '../middlewares/partyMiddleware';
+
+const router = express.Router();
+
+const apiStr = '/api/v1/parties';
```

Comment:
@frankly034 let the naming of this reflect what value it's holding, what do you think about doing `const apiUrl` and also ensure to version your API.

Reply:
@Lumexralph Thanks for the observation, was actually looking for a very descriptive name. I've made the changes and pushed